.github/workflows/ci.yml

```diff
@@ -27,50 +30,67 @@ jobs:
   test:
     name: python${{ matrix.python-version }}-${{ matrix.platform.python-architecture }} ${{ matrix.platform.os }}
     runs-on: ${{ matrix.platform.os }}
+    needs: [lint, check-msrv, linalg-example]
     strategy:
-      max-parallel: 16
```

Comment:
Are jobs still parallelized without this? I haven't used actions recently so sorry if I'm asking something outdated

Reply:
Yes, they still run in parallel. But I would not have known, I had to try.